internal/web3ext/web3ext.go

```diff
@@ -532,13 +532,13 @@ web3._extend({
 	property: 'shh',
 	methods: [
 		new web3._extend.Method({
-			name: 'setMaxMessageLength',
-			call: 'shh_setMaxMessageLength',
+			name: 'setMaxMessageSize',
+			call: 'shh_setMaxMessageSize',
 			params: 1
 		}),
 		new web3._extend.Method({
-			name: 'setMinimumPoW',
-			call: 'shh_setMinimumPoW',
+			name: 'setMinPoW',
+			call: 'shh_setMinPoW',
 			params: 1
```

Comment:
web3etx is reserved for functionality not offered by web3. Now that web3 has whisper support most or maybe all ext shh methods/properties can be removed.

Reply:
That's good to know, thanks. I have checked which functions are indeed present, and it seems that only `getFilterMessages`, `deleteMessageFilter` and `publicKey` are missing. I'm waiting for Fabian to discuss the web3.js update, but if I can add those functions then indeed they could be removed from this file.

Reply:
@bas-vk so actually, in this repo the web3.js doesn't work (it uses the http rpc interface that needs to be started explicitly), and it's version 0.20.1 which doesn't work anyway. I wanted to upgrade to 1.0.0 but Fabian says that this won't work. So we still need the web3ext functions for now.

Reply:
Alright, I managed to get 0.20.1 to work, so I updated it and now the api does work, I am able to send and receive messages. So as you suggested, @bas-vk , I removed the web3ext functions.